tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -671,8 +670,19 @@ docker:
 EOT
 	fi
 
+	if [ "${UMOCI}" = "yes" ]; then
+		umoci_url="$(get_package_version_from_kata_yaml externals.umoci.url)"
+		umoci_tag="$(get_package_version_from_kata_yaml externals.umoci.tag)"
+		info "Install umoci"
+		git clone "${umoci_url}" --branch "${umoci_tag}"
+		pushd umoci
+		make
+		install -o root -g root -m 0755 umoci "${ROOTFS_DIR}/usr/local/bin/"
+		popd
+	fi
+
 	if [ -n "${AA_KBC}" ]; then
-		[ -z "${SKOPEO_UMOCI}" ] && die "SKOPEO_UMOCI must be set to install attestation-agent"
+		[ "${UMOCI}" != "yes" ] && die "UMOCI must be set to install attestation-agent"
```

Comment:
Just to simplify things, wouldn't be better to force UMOCI installation and print a giant warning saying "UMOCI must be set, it wasn't, I'm installing it, bye!" in this specific case? :-)

I mean, if I were to be enabling the AA_KBC, there's a big chance I could forget enabling UMOCI or even take that as that's a dependency it'd automatically set UMOCI="yes" in the scripts.

Reply:
That sounds good to me. If @Jakob-Naucke is happy I can implement this change

Reply:
@fidencio good idea, @stevenhorsman sure go ahead :)